dashboard/app/models/ability.rb

```diff
@@ -106,7 +106,7 @@ def initialize(user)
 
       if user.workshop_organizer?
         can :create, Pd::Workshop
-        can [:read, :start, :end, :update, :destroy], Pd::Workshop, organizer_id: user.id
+        can [:read, :start, :end, :update, :destroy, :workshop_organizer_survey_report], Pd::Workshop, organizer_id: user.id
```

Comment:
I'm not sure this is what you want. This is adding a new action `workshop_organizer_survey_report` to `Pd::Workshop`. 

In any case, as I said above, it doesn't look like any auth is being done (though it should be). Maybe you want to do it this way and authorize it somewhere in the controller. Though I think it makes more sense to make the survey report the object and have read be the verb / action. You can use symbols instead of classes, something like:

```ruby
can :read, :workshop_organizer_survey_report
```